src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -73,11 +77,14 @@ private class Indexing(resources: Resources[Task], cache: Caches[Task], coordina
     case Right(value)                => Right(value)
   }
 
-  def startKafkaStream(): Unit = {
+  private def send(request: HttpRequest): Future[HttpResponse] =
```

Comment:
need to update the request to put an `OAuth2BearerToken` to have access to the the stream

Reply:
and also to log an error if the we can a 401 or 403